lib/openshift_client.rb

```diff
@@ -52,5 +52,16 @@ def initialize(uri,
     def all_entities
       retrieve_all_entities(ENTITY_TYPES)
     end
+
+    def process_template(template)
```

Comment:
@simon3z yes, the parameters are passed as part of the template. In order to process a template one should pass a template that contains objects list and the desirable (and required) parameters as in the test below.